components/more_channels/more_channels.tsx

```diff
@@ -68,15 +84,15 @@ export default class MoreChannels extends React.PureComponent {
         if (this.props.bodyOnly) {
             this.handleExit();
         }
-    }
+    };
```

Comment:
We don't require semi-colons at the end of these arrow functions, just for the future.